crates/macros/src/lib.rs

```diff
@@ -90,6 +90,7 @@ pub fn build(stream: proc_macro::TokenStream) -> proc_macro::TokenStream {
                                 if let ::std::option::Option::Some(filename) = path.file_name() {
                                     if filename == profile {
                                         copy(source, &mut path);
+                                        println!("cargo:rustc-link-search=native={}", path.to_str().expect("Invalid profile dir"));
```

Comment:
This is already supported. See https://github.com/microsoft/windows-app-rs/ for an example.

Reply:
The `cargo:rustc-link-search=native=...` output specifically? Or do you mean static libs in general?

Reply:
Sorry, just the link search.

Reply:
Throw your libs into `.windows/[arch]`, with [arch] being x86, x64, or arm64 (see below). That path is automatically added to the link search path.

Real world example: https://github.com/microsoft/windows-app-rs/tree/master/.windows

Impl for reference:
https://github.com/microsoft/windows-rs/blob/d3870edf5bdc39e14af4d4144015626c959ad7b8/crates/macros/src/lib.rs#L108-L117

Reply:
Cool, I'll undo that part.